Essentials/src/com/earth2me/essentials/Settings.java

```diff
@@ -1653,6 +1653,12 @@ public boolean logCommandBlockCommands() {
         return blacklist;
     }
 
+    // #easteregg
+    @Override
+    public String getContextPrefix() {
+        return config.getString("context-prefix", "");
```

Comment:
Just want to note my objection here to defaulting to no namespacing! :)

Reply:
Noted! I went and explained my reasoning below. The issue of default namespacing can be tackled at a later date, assuming that it becomes a real issue.